nx-X11/programs/Xserver/os/log.c

```diff
@@ -561,15 +561,6 @@ VAuditF(const char *f, va_list args)
     prefix = AuditPrefix();
     len = vsnprintf(buf, sizeof(buf), f, args);
 
-#if 1
-    /* XXX Compressing duplicated messages is temporarily disabled to
-     * work around bugzilla 964:
-     *     https://freedesktop.org/bugzilla/show_bug.cgi?id=964
```

Comment:
> The code is reenabled without change, which according to the bugzilla entry might cause 
> problems leading to 100% CPU consumption. Apparently in the unmaintained xprint, but it seems
> worth noting. Perhaps danger of DoS?

The issue occurred in Xprt which has been fully removed from nx-libs. I have reenabled this as done in X.org Xserver after Xprt removal. Valid point, though.